practice-one/src/components/mobile-app-introduction.html

```diff
@@ -0,0 +1,24 @@
+<section class="mobile-app-introduction-container">
+  <div class="introduction-app-container">
+    <h2 class="introduction-app-heading">
+      It's Now 
+      <span class="primary-color">More Easy</span>
+      to 
+      <span class="accent-color">Order</span> 
+      by Our Mobile 
+      <span class="primary-color">App</span>
+    </h2>
+    <p class="introduction-app-content">This is a type of resturent which typically serves food and drink, in addition to light refreshments such as baked goods or snacks. The term comes frome the rench word meaning food</p>
+    <div class="app-icon">
+      <div class="play-store">
+        <img src="./assets/images/play-store.svg" alt="Dowload in play store">
+      </div>
+      <div class="app-store">
+        <img src="./assets/images/app-store.svg" alt="Dowload in app store">
+      </div>
```

Comment:
Recheck these tag, should these be button?

Reply:
That's better than div tag, thanks for your comment